green_city/tests/api/get_eco_news_dislikes_test.py

```
@@ -0,0 +1,92 @@
+import pytest
+import requests
+
+from green_city.src.config import API_BASE_URL_8085, ENDPOINTS
+
+
+def test_get_eco_news_dislikes_count_success(create_news, auth_token):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)